distribution/std-lib/Standard/src/Base/Data/Json.enso

```diff
@@ -76,44 +112,78 @@ type Json
         Json.Null -> Nothing
         Json.Object f -> f.map .unwrap
 
-## A failure indicating malformed text input into the JSON parser.
+## UNSTABLE
+
+   A failure indicating malformed text input into the JSON parser.
 
    Check the `message` field for detailed information on the specific failure.
 type Parse_Error message
 
-## Converts the error to a display representation.
+## UNSTABLE
+
+   Converts the error to a display representation.
 Parse_Error.to_display_text : Text
 Parse_Error.to_display_text = "Parse error in parsing JSON: " + this.message.to_text + "."
 
-## Gets the value associated with the given key in this object. Returns
-   `Nothing` if the associated key is not defined.
-Object.get : Text -> Json | Nothing
+## Gets the value associated with the given key in this object.
+
+   Arguments:
+   - field: The name of the field from which to get the value.
+
+   Throws `Nothing` if the associated key is not defined.
+Object.get : Text -> Json ! Nothing
 Object.get field = this.fields.get field
 
 ## Parses an RFC-8259 compliant JSON text into a `Json` structure.
+
+   Arguments:
+   - json_text: The RFC-8259-compliant JSON text.
+
+   > Example
+     Convert a JSON object into JSON.
```

Comment:
`Convert a JSON object into JSON.`
This is IMO unclear, it doesn't say that we are converting a textual-representation of the JSON, so it sounds a bit like 'convert JSON into JSON'.